packages/ignite/src/commands/doctor.js

```diff
@@ -1,49 +1,82 @@
-  const ignitePath = R.trim(Shell.which('ignite') || '')
-  const igniteVersion = R.trim(Shell.exec('ignite version', { silent: true }).stdout)
-  const nodePath = R.trim(Shell.which('node'))
-  const npmVersion = R.trim(Shell.exec('npm --version', { silent: true }).stdout)
-  const npmPath = R.trim(Shell.which('npm'))
-  const nodeVersion = R.trim(Shell.exec('node --version', { silent: true }).stdout)
-  const ggPackageFile = `${process.cwd()}/node_modules/gluegun/package.json`
-  const ggVersion = exists(ggPackageFile) ? read(ggPackageFile, 'json').version : '¯\\_(ツ)_/¯'
-
-  const rnCli = R.split(/\s/, R.trim(Shell.exec('react-native --version', { silent: true }).stdout))[1] // lulz
-
-  const rnPackageFile = `${process.cwd()}/node_modules/react-native/package.json`
-  const appReactNativeVersion = Shell.test('-f', rnPackageFile) ? require(rnPackageFile).version : '¯\\_(ツ)_/¯'
-
-  const body = `
-\`\`\`
-Computer
-  Platform: ${platform}
-Ignite
-  Version: ${igniteVersion}
-  Path: ${ignitePath}
-Node
-  Version: ${nodeVersion}
-  Path: ${nodePath}
-NPM
-  Version: ${npmVersion}
-  Path: ${npmPath}
-Gluegun
-  Version: ${ggVersion}
-React Native CLI
-  Version: ${rnCli}
-App
-  React Native Version: ${appReactNativeVersion}
-\`\`\`
-`
-  debug(body, 'Ignite Info')
+  const arch = os.arch()
+  const cpus = os.cpus() || []
+  const firstCpu = head(cpus) || {}
+  const cpu = `${firstCpu.model}`
+  const cores = `${cpus.length} cores`
+  const ignitePath = which('ignite')
+  const igniteVersion = await run('ignite version', { trim: true })
+  const nodePath = which('node')
+  const npmVersion = await run('npm --version', { trim: true })
+  const npmPath = which('npm')
+  const nodeVersion = replace('v', '', await run('node --version', { trim: true }))
+  const rnCli = split(/\s/, await run('react-native --version', { trim: true }))[1] // lulz
+  const xcodeVersion = split(/\s/, await run('xcodebuild -version', { trim: true }))[1] // lulz
+  const rnPkg = read(`${process.cwd()}/node_modules/react-native/package.json`, 'json')
+  const appReactNativeVersion = rnPkg ? rnPkg.version : '-'
+  const androidPath = process.env['ANDROID_HOME']
+  const javaVersionCmd = process.platform === 'win32' ? 'java -version' : 'java -version 2>&1'
+  const javaVersion = last(match(/"(.*)"/, await run(javaVersionCmd)))
+  const javaPath = which('java')
+
+  // display helpers
+  const column1 = (label, length = 16) => padEnd(label, length)
+  const column2 = label => colors.yellow(padEnd(label, 10))
+  const column3 = label => colors.muted(label)
+
+  // print ignite
+  header()
+  info('')
+
+  info(colors.cyan('System'))
+  table([
+    [column1('platform'), column2(platform)],
+    [column1('arch'), column2(arch)],
+    [column1('cpu'), column2(cores), column3(cpu)]
+  ])
+
+  info('')
+  info(colors.cyan('JavaScript'))
+  table([
+    [column1('node'), column2(nodeVersion), column3(nodePath)],
+    [column1('npm'), column2(npmVersion), column3(npmPath)]
```

Comment:
we should probably check if they have `yarn`, and if so what version, since our CLI will try to use Yarn and yarn has had quite a few bugs.